p2p/key.go

```diff
@@ -65,26 +76,23 @@ func LoadNodeKey(filePath string) (*NodeKey, error) {
 	nodeKey := new(NodeKey)
 	err = cdc.UnmarshalJSON(jsonBytes, nodeKey)
 	if err != nil {
-		return nil, fmt.Errorf("error reading NodeKey from %v: %v", filePath, err)
+		return nil, err
```

Comment:
Might be nice to keep the errorf, otherwise the user will only get an "invalid JSON" error without any information about which file has invalid JSON.

Reply:
shouldn't it be responsibility of a caller? it provides filePath

Reply:
I suppose.